src/app/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -229,7 +233,20 @@ export const PlanActionsDropdown: React.FunctionComponent<IPlansActionDropdownPr
         mutateResult={deletePlanMutation}
         title="Permanently delete migration plan?"
         confirmButtonText="Delete"
-        body={`All data for migration plan "${plan.metadata.name}" will be lost.`}
+        body={
+          isPlanStarted && !isPlanArchived ? (
+            <TextContent>
+              <Text>
+                Migration plan &quot;{plan.metadata.name}&quot; will be deleted. However, some
```

Comment:
I think this could be more concise. Suggestion:

Migration plan &quot;{plan.metadata.name}&quot; will be deleted. However, deleting a migration plan does not remove temporary resources such as failed VMs and data volumes, conversion pods, importer pods, secrets, or config maps.

To clean up these resources, archive the plan before deleting it.